modules/rtticketedit.php

```diff
@@ -142,7 +142,7 @@
                 break;
             case 'resetpriority':
                 $ticket = $LMS->GetTicketContents($id);
-                if ($ticket['priority'] != RT_PRIORITY_NORMAL) {
+                if (!is_null($ticket['priority']) && $ticket['priority'] != RT_PRIORITY_NORMAL) {
```

Comment:
> !is_null($ticket['priority'])

A to nie to samo co `issset($ticket['priority'])`?

Reply:
Tak - to dobre uproszczenie.